### ISSUE TITLE
Cannot insert an image into CKEditor when editing a custom block through Layouts

### ISSUE THREAD

**Issue by quicksketch** [backdrop/backdrop-issues#1125]
As originally mentioned in https://github.com/backdrop/backdrop-issues/issues/1088#issuecomment-134862039 by @Graham-72, we have an issue where you cannot edit/insert an image in the Layouts UI on a custom block.

To reproduce:

- Add a custom block under Structure > Custom blocks > Add
- Edit a layout and add this custom block to the layout.
- While configuring the block, try to insert an image.
- When hitting "Save" on the dialog, the dialog will not close and the image is not inserted.

This seems to be caused by the fact that opening the image dialog *closes* the configure dialog. This is likely because these are both "modal" dialogs, of which Backdrop can only support one at a time. When one is opened, the other is replaced. We need to make either the image (or Layout) dialog not use a modal. Or make it so that multiple modals can be present.

For clarification on terminology, a "modal" dialog disables the area behind it until completed or dismissed. A normal dialog may usually be moved around and does not disable the page behind it. Currently Layout and Image dialogs are modals, which the Token insert interface is a normal dialog.

**Comment by quicksketch** [comment]
PR filed at https://github.com/backdrop/backdrop/issues/1037 that should fix this issue. I gave CKEditor its own dedicated dialog and selector so that it won't conflict with other modules that use the built-in modal. As this was the first time we tried nested modals, I found another issue with binding and unbinding the resize event. Now when a dialog is closed it correctly unbinds from the resize event.
